.github/workflows/documentation-links.yml

```
@@ -0,0 +1,21 @@
+name: Read the Docs PR preview
+on:
+  pull_request_target:
+    types:
+      - opened
+    paths:
+    - 'Doc/**'
+    - 'Misc/**'
+    - '.github/workflows/doc.yml'
+
+permissions:
+  pull-requests: write
+
+jobs:
+  documentation-links:
```

```suggestion
  pr-link:
    name: Add link to PR
```

Maybe clarify this a bit?

---

Can do, I kept the same name already used in the other repos.

---

Oh, in that case may as well be consistent I guess.